Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIStorageCocoa.mm

```diff
@@ -34,13 +34,18 @@
 
 #import "CocoaHelpers.h"
 #import "WebExtensionConstants.h"
+#import "WebExtensionContextProxy.h"
+#import "WebExtensionContextProxyMessages.h"
 #import "WebExtensionStorageAccessLevel.h"
 #import "WebExtensionStorageType.h"
 #import "WebExtensionUtilities.h"
 #import "_WKWebExtensionStorageSQLiteStore.h"
 #import <wtf/BlockPtr.h>
 #import <wtf/cocoa/VectorCocoa.h>
 
+static NSString * const newValueKey = @"newValue";
+static NSString * const oldValueKey = @"oldValue";
```

Comment:
It looks like these keys are only used in `fireStorageChangedEventIfNeeded()`, if so you could move them inside that function scope.